zeppelin-web/package.json

```diff
@@ -2,31 +2,42 @@
   "name": "zeppelin-web",
   "license": "Apache-2.0",
   "version": "0.0.0",
-  "engines" : { "node" : ">=6.0.0" },
+  "engines": {
+    "node": ">=0.10.0"
+  },
   "scripts": {
     "postinstall": "bower install --silent && grunt googlefonts",
-    "build": "grunt build",
-    "start": "grunt serve",
-    "test": "grunt test",
-    "pretest": "npm install karma-phantomjs-launcher"
+    "build": "grunt pre-webpack-dist && webpack && grunt post-webpack-dist",
+    "dev:server": "webpack-dev-server --hot",
+    "dev:watch": "grunt watch-webpack-dev",
+    "prestart": "grunt pre-webpack-dev",
+    "start": "npm-run-all --parallel dev:server dev:watch",
```

Comment:
Thanks for review :) @AhyoungRyu

> Since there is default command `npm start` ....
> How about `dev` instead of `start`

So, you mean `start:server` and `start:watch`. right?